app/services/questionnaire_responses/monthly_screening_reports.rb

```diff
@@ -21,10 +23,6 @@ def pre_fill
 
   private
 
-  def month_date_str
```

Comment:
Moved 3 methods, which were reused by form initializing services to a common helper module.